data_source_transip_private_network_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccTransipDataSourcePrivateNetwork(t *testing.T) {
+	var testConfig = `data "transip_private_network" "test" {name = "%s"}`
```

Comment:
Please add a skip condition for when `TF_VAR_private_network_name` is not provided, like: https://github.com/aequitas/terraform-provider-transip/pull/16/files#diff-9b7b3d4debe01c7dce8fa350cddaed83R12-R15.